File: example/scalalib/testing/5-test-stealing/foo/test/src/RandomTestsD.scala
package foo
import utest._
object RandomTestsD extends RandomTestsUtils {
  def tests = Tests {
    test("test1") { testGreeting("Kai", 14) }
    test("test2") { testGreeting("Luna", 11) }
    test("test3") { testGreeting("Mason", 15) }
    test("test4") { testGreeting("Nova", 10) }
    test("test5") { testGreeting("Owen", 13) }
    test("test6") { testGreeting("Piper", 12) }
    test("test7") { testGreeting("Quinn", 14) }
    test("test8") { testGreeting("River", 11) }
  }
} 

File: example/kotlinlib/testing/5-test-stealing/foo/test/src/foo/RandomTestsD.kt
package foo

import org.junit.jupiter.api.Test

class RandomTestsD : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() { testGreeting("Kai", 14) }
    
    @Test
    @Throws(Exception::class)
    fun test2() { testGreeting("Luna", 11) }
    
    @Test
    @Throws(Exception::class)
    fun test3() { testGreeting("Mason", 15) }
    
    @Test
    @Throws(Exception::class)
    fun test4() { testGreeting("Nova", 10) }
    
    @Test
    @Throws(Exception::class)
    fun test5() { testGreeting("Owen", 13) }
    
    @Test
    @Throws(Exception::class)
    fun test6() { testGreeting("Piper", 12) }
    
    @Test
    @Throws(Exception::class)
    fun test7() { testGreeting("Quinn", 14) }
    
    @Test
    @Throws(Exception::class)
    fun test8() { testGreeting("River", 11) }
} 